package.json

```diff
@@ -10,6 +10,7 @@
   "bin": {
     "axe-storybook": "bin/axe-storybook.js"
   },
+  "type": "module",
```

Comment:
Will output ESM instead of CJS now. Needed because the Storybook packages now export ESM, and this makes it a lot easier to consume them.

Been meaning to make this change anyway, and the versions of Node we support (18+) already support ESM.